### ISSUE TITLE
[Bug] Tony Hawk's Pro Skater 1+2 crash to desktop on Ubuntu

### ISSUE THREAD

**Issue by Kupo91** [Ryujinx/Ryujinx#5128]
### Description of the issue

If I try to launch Tony Hawk's Pro Skater 1+2, the game starts with the logo from the publisher. After like two seconds the emulator will crash to desktop. I don't have this issue on my Windows machine.

### Reproduction steps

Launch the game on Ubuntu 22.04, wait for the first video animation.

### Log file

[Ryujinx_1.1.829_2023-05-28_17-11-20.log](https://github.com/Ryujinx/Ryujinx/files/11584970/Ryujinx_1.1.829_2023-05-28_17-11-20.log)

### OS

Ubuntu 22.04

### Ryujinx version

1.1.829

### Game version

1.0.3

### CPU

i9-11900KB

### GPU

NVIDIA GeForce GTX 1050 Ti

### RAM

16GB

**Comment by gdkchan** [comment]
Sounds like a duplicate of #3372, but I didn't test. I would recommend trying the suggestions there to confirm if its the same issue or not.

**Comment by piplup55** [comment]
> Sounds like a duplicate of #3372, but I didn't test. I would recommend trying the suggestions there to confirm if its the same issue or not.

seems to be related it's a ue4 game, so likely just needs vm max count to be higher

**Comment by Kupo91** [comment]
Ah, I searched for issues with this game beforehand but couldn't finy anything. 
So setting memory management from "Host (Unchecked)" to "Software" fixed the issue. Atleast for a moment xD I was able make it to the start screen with this setting. When it said "Press A" it crashed again though.

New log: 
[Ryujinx_1.1.829_2023-05-28_18-14-06.log](https://github.com/Ryujinx/Ryujinx/files/11585125/Ryujinx_1.1.829_2023-05-28_18-14-06.log)
